charts/pro-ui/values.yaml

```
@@ -265,8 +265,16 @@ hpa:
   targetCPU: 100
   targetMemory: ''
 
-# @section Artifacts Storage settings
+# @param pdb.enabled If PDB is enabled for the service.
+# @param pdb.minAvailable How many pods must be available after the eviction.
+# @param pdb.maxUnavailable How many pods can be unavailable after the eviction.
+# if minAvailable and pdb.maxUnavailable are both set, only minAvailable is used
+pdb:
+  enabled: false
+  minAvailable: 1
+  maxUnavailable: 0
```

давайте сделаем по аналогии с другими чартами:
```suggestion
  minAvailable: ''
  maxUnavailable: 1
```